Makefile

```diff
@@ -1,5 +1,9 @@
+ghcid: dev
+dev: ## Start ghcid
+	@ghcid --target lib:matchmaker --allow-eval --warnings -o ghcid.text
```

Comment:
What is the difference between `-o ghcid.text | tail -f` and simply running ghcid on stdout?

Reply:
I'm not sure what the `-o ghcid.text` bit is doing here. I just left it as is assuming someone else wanted it that way.

Reply:
hmm you're right, actually I think it was @JonathanLorimer who introduced it initially

Reply:
it just outputs ghcid to a file. I have always used it, but not sure what the advantage is.

Reply:
Okay, then I vote that we remove it for the time being. We can introduce a `$GHCID_OPTS` variable if it's really needed during day-to-day development.

Reply:
Okay I'll remove it

Reply:
Great, don't forget to squash your commits when merging. :)